.changes/1347-new-update-features.md

```diff
@@ -0,0 +1,3 @@
+- [new-general] : User experience enhancement on News Update
+- [new-general] : Now user can post news with multiple slides
+- [new-desktop] : Desktop have now support of adding news update
```

Comment:
Thanks for moving the file. Didn't read the update before thus I'd suggest to update it in this PR to be more informative for actual users:
```suggestion
- [Updates] : Completely revamped editor for creating new Updates with support for multiple slides per update, Video and Image posting and a refreshed text editor
```